src/views/transaction/components/transaction-overview/transaction-overview.view.jsx

```diff
@@ -98,6 +99,8 @@ function TransactionOverview ({
         return onExit(amount, account, fee)
       }
       default: {
+        setIsButtonDisabled(true)
```

Comment:
You would need to do this at the start of the function, or it will only be applied for transfers

Reply:
Makes sense.